node/config/def.go

```diff
@@ -83,6 +83,7 @@ func DefaultFullNode() *FullNode {
 		Fevm: FevmConfig{
 			EnableEthRPC:                 false,
 			EthTxHashMappingLifetimeDays: 0,
+			EthTraceFilterMaxResults:     100,
```

Comment:
Is `100` a good default for this? I don't know the nature of the types of responses returned from this API or the typical usage. It does seem like it's going to be more expensive than something like `eth_getLogs` where we can spit thousands of results at you fairly cheaply because in this case we may have to execute tipsets to get the traces. But that's cached, so maybe it's not all that expensive to do 1000 vs 100? Do you have any hunch of what users might expect from this API?